views/game.nunjs

```diff
@@ -35,17 +35,20 @@
         }
     </style>
     <script src="https://cdnjs.cloudflare.com/ajax/libs/socket.io/2.1.1/socket.io.js"></script>
-    <script src="/static/js/planetwars.js"></script>
+    <script src="/static/planetwars.js"></script>
     <script>
         function main() {
             var gameID = "{{ gameID }}";
             var socket = io.connect("/game");
             var canvas = document.getElementById("game");
             var ctx = canvas.getContext("2d");
-            socket.on("initialize", function(msg) {
-                var data = JSON.parse(msg);
+
+            socket.on("initialize", function(messagedata) {
+		console.log(messagedata);
+                var data = messagedata
```

Comment:
Just call the arg `data` instead of `messagedata`, or `msg` for consistency with below.